src/utils/money.cljs

```diff
@@ -42,25 +42,35 @@
   [x]
   (instance? BigNumber x))
 
+(defn ->bignumber
+  [n]
+  (if (bignumber? n) n (bignumber n)))
+
+(defn ->bignumbers
+  [& nums]
+  (let [bignums (map ->bignumber nums)]
+    (when (every? bignumber? bignums)
+      bignums)))
+
 (defn greater-than-or-equals
-  [^js bn1 ^js bn2]
-  (when (bignumber? bn1)
+  [^js n1 ^js n2]
+  (when-let [[^js bn1 ^js bn2] (->bignumbers n1 n2)]
     (.greaterThanOrEqualTo bn1 bn2)))
 
 (defn greater-than
-  [bn1 bn2]
-  (when (bignumber? bn1)
+  [n1 n2]
+  (when-let [[^js bn1 ^js bn2] (->bignumbers n1 n2)]
     (.greaterThan ^js bn1 bn2)))
 
 (defn less-than
-  [bn1 bn2]
-  (when (bignumber? bn1)
+  [n1 n2]
+  (when-let [[^js bn1 ^js bn2] (->bignumbers n1 n2)]
     (.lessThan ^js bn1 bn2)))
 
 (defn equal-to
   [n1 n2]
-  (when-let [^js bn1 (bignumber n1)]
-    (.eq ^js bn1 n2)))
+  (when-let [[^js bn1 ^js bn2] (->bignumbers n1 n2)]
+    (.eq ^js bn1 bn2)))
```

Comment:
Here we're adding some additional checks for big number instances when using our big number utility functions. Let me what you think 🙏

Reply:
I love this approach!